WordPress/src/main/res/layout/activity_log_list_item_menu_item.xml

```diff
@@ -0,0 +1,28 @@
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="@dimen/menu_item_height"
+    android:gravity="start|center_vertical"
+    android:orientation="horizontal"
+    android:paddingStart="@dimen/menu_item_margin"
+    android:paddingEnd="@dimen/menu_item_margin">
+
+    <ImageView
+        android:id="@+id/image"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:contentDescription="@null"
+        android:tintMode="multiply"
+        android:layout_marginEnd="@dimen/margin_large"
+        tools:src="@drawable/ic_history_white_24dp"
+        tools:tint="@color/primary" />
+
+    <org.wordpress.android.widgets.WPTextView
```

Comment:
We might want to use material text view in the new layout.

Reply:
@ashiagr - thanks and yes, I'll add that to my task list.